src/views/index.ejs

```diff
@@ -0,0 +1,158 @@
+								<polygon points="2,21 22,21 14,11.5 14,5 10,3 10,11.5"></polygon>
+							</svg>
+							<h2>Beta</h2>
+						</label>
+						<% } %>
+						<% if(PNID.access_level === 3) { %>
+						<input type="radio" id="dev" name="server_selection" value="dev" <% if (PNID.server_access_level === 'dev') { %> checked="" <% } %> <% if(PNID.access_level < 3) { %> disabled <% } %> >
+						<label for="dev">
+							<svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
+								<polygon points="2,21 22,21 14,11.5 14,5 10,3 10,11.5"></polygon>
+							</svg>
+							<h2>Dev</h2>
+						</label>
+						<% } %>
+                    </div>
+                </div>
+                <div class="group">
+                    <h2>Email Notifications</h2>
+                    <div class="radio server-selection" id="server">
+                        <input type="radio" id="true" name="marketing_flag" value="true" <% if (PNID.flags.marketing) { %>checked=""<% } %>>
+                        <label for="marketing_flag">
+                            <h2>Receive</h2>
+                        </label>
+                            <input type="radio" id="false" name="marketing_flag" value="false" <% if (!PNID.flags.marketing) { %>checked=""<% } %>>
+                        <label for="marketing_flag">
+                            <h2>Do Not Receive</h2>
+                        </label>
+                    </div>
+                </div>
+                <div class="group">
+                    <h2>Non-Nintendo Device Setting</h2>
+                    <div class="radio server-selection" id="server">
+                        <input type="radio" id="true" name="off_device_flag" value="true" <% if (PNID.flags.off_device) { %>checked=""<% } %>>
+                        <label for="off_device_flag">
+                            <h2>Allow</h2>
+                        </label>
+                            <input type="radio" id="false" name="off_device_flag" value="false" <% if (!PNID.flags.off_device) { %>checked=""<% } %>>
+                        <label for="off_device_flag">
+                            <h2>Deny</h2>
+                        </label>
+                    </div>
+                </div>
+            </div>
+        </form>
+        <% if (notice) { %>
+        <script>
+                wiiuDialog.alert('<%= notice %>', 'OK');
+        </script>
+        <% } %>
+		<script>
+			var seen = false;
+			function warning() {
+				if(seen) return;
+				seen = true;
+				wiiuDialog.alert('Warning: Changing your country may cause unexpected results. You may need to re-link your PNID for the changes to apply. \n\nFor more information, see:\nhttps://pretendo.network/docs/errors/PAS-727-0001', 'I Understand');
```

Comment:
This link doesn't exist. Also we now own `preten.do` for short links, do you want to use that instead?